activity/activity_GeneratePDFCovers.py

```
@@ -62,7 +62,7 @@ def do_activity(self, data):
             self.logger.error(error_message)
             self.emit_monitor_event(self.settings, article_id, version, run,
                                     self.pretty_name, "error", error_message)
-            return self.ACTIVITY_PERMANENT_FAILURE
+            return self.ACTIVITY_SUCCESS
```

this will ignore any error, including the service being completely broken. Can't we do anything better?

---

A very simple option is we could return a `ACTIVITY_TEMPORARY_FAILURE` which would result in retying this activity every 10 seconds until the workflow ultimately reaches `Timed Out` status. That would be slightly better than it resulting in `Failed` to me. The covers that will never be generated will always result in `Timed Out`.

I'll add more details of other options to consider into issue https://github.com/elifesciences/issues/issues/5364.